src/letter_counts.py

```diff
@@ -0,0 +1,17 @@
+def letter_freq(txt):
+    # using dictonary approach to get count
+    # of each element in string
+    freqs = {}
+
+    for keys in txt:
+        freqs[keys] = freqs.get(keys, 0) + 1
```

Comment:
please use the argument name for optional arguments. Makes it a bit easier to understand:
`freqs.get(keys, default=0)`
Otherwise i really like this solution!